internal/policies/mount/testdata/TestWriteMountsFile/write_mounts_file_with_values_separated_by_','/mounts

```diff
@@ -0,0 +1,5 @@
+protocol://domain.com/mount-0-path
```

Comment:
Also, the issue with that generation is that they are always ascii ordered,which results in ascii order in the golden files and might give the false impression we are ordering entries. So the generation needs to be a little bit smarter to avoid this.